service.subtitles.titlovi-com-official/addon.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.subtitles.titlovi-com-official"
+       name="Titlovi.com Official "
+       version="1.2.0"
+       provider-name="titlovi.com">
```

Comment:
Are you part of titlovi.com or just an external developer? If the second please set yourself as an author after titlovi.com

Reply:
Fixed